### PR TITLE
51 - [Homepage] Style h3s with line

### LINKED ISSUE #51: [Homepage] Style h3s with line
We currently have the sidebar `h3`s  styled with lines:
https://github.com/hlxsites/servicenow/blob/main/scripts/scripts.js#L194-L198
https://github.com/hlxsites/servicenow/blob/main/styles/styles.css#L167-L175

We should style the rest of the h3s from the homepage (based on the template metadata) to have the same appearance.
We could generalise the code to use a CSS class and not look for descendants of the sidebar only.

### PR DESCRIPTION
**Pull request by bala-servicenow**
Fix #51 

Test URLs:
- Before: https://main--servicenow--hlxsites.hlx.live/
- After: https://fixes-issue51--servicenow--hlxsites.hlx.live/blogs

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -292,15 +292,19 @@ async function detectSidebar(main) {
...
-
-    sidebar.querySelectorAll('h3').forEach((header) => {
-      const headerContent = header.textContent;
-      header.textContent = '';
-      header.append(span(headerContent));
-    });
   }
 }
 
+async function h3Styling(main) {
```
Comment (andreituicu) [inline scripts/scripts.js:298]: Does it have to be async?

File: scripts/scripts.js [hunk 2]
```diff
@@ -337,6 +341,7 @@ async function loadEager(doc) {
     decorateMain(main);
     await loadEagerBlocks(main);
     await detectSidebar(main);
+    await h3Styling(main);
```
Comment (andreituicu) [inline scripts/scripts.js:344]: Usually we put these kind of functions at the end of `decorateMain`, unless there is a good reason for them to be outside.

File: scripts/scripts.js (hunk 1 above)
Comment (bala-servicenow) [reply scripts/scripts.js:298]: Yes, first i tried without async, threw error as the elements were not available on the page.

Comment (bala-servicenow) [reply scripts/scripts.js]: moving it to decorate main and removing async works fine. Updated the code.

File: scripts/scripts.js (hunk 2 above)
Comment (bala-servicenow) [reply scripts/scripts.js]: moved the code to decorateMain, working fine.

**Review by andreituicu — APPROVED** [review]
LGTM! 🎉

**Comment by andreituicu** [comment]
@bala-servicenow you can merge it when it's ok for you.

**Pull request merged**